swagger_server/database.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+from configparser import ConfigParser, ExtendedInterpolation
+from sqlalchemy import create_engine
+from swagger_server import ons_logger
+
+
+def drop(config, engine, logger):
+    db_connection = config.get('db_connection')
+    db_schema = config.get('db_schema')
+
+    from swagger_server.models_local.base import Base
+    # fix-up the postgres schema:
+    Base.metadata.schema = db_schema if db_connection.startswith('postgres') else None
+
+    from swagger_server.models_local import _model
```

Comment:
Fixed. This is likely to change again in a subsequent PR as ras-party-v2 drifts away from code generator.